packages/ui/src/cairo_alpha/MacrosSection.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+  import type { MacrosOptions } from '@openzeppelin/wizard-cairo-alpha';
+  import HelpTooltip from '../common/HelpTooltip.svelte';
+
+  export let macros: MacrosOptions;
+</script>
+
+<section class="controls-section">
+  <h1>Macros</h1>
+
+  <div class="checkbox-group">
+    <label class:checked={macros.withComponents}>
+      <input type="checkbox" bind:checked={macros.withComponents} />
+      with_components
+      <HelpTooltip link="https://docs.openzeppelin.com/contracts-cairo/alpha/macros/with_components">
+        Use the "with_components" macro for simplified contract structure.
```

Comment:
This can use html if desired:
```suggestion
        Use the <code>with_components</code> macro for simplified contract structure.
```